py/examples/device_tapdetector_example/README.md

```diff
@@ -0,0 +1,31 @@
+## Tap Detector
+### About
+The tap detector is designed to take input from two differnt sources of data and to "match" them together by detecting taps between the two devices. For example, two devices recording at the same time can be tapped against each other. Then, the resulting data files from both devices can be used with the tap detector to find the corresponding taps in either file so that their timestamps can be aligned. Data smoothing is done using a hann filter, whose window is specified by the user for each source of data.
+
+Sample data is included in a .zip file to use. The following command/parameters works well for the sample data provided. Note that the sample data includes noise at the end.
+```py .\py\emotibit\device_tapdetector.py -sof .\emotibit5_AX.csv .\emotibit5_AY.csv .\emotibit5_AZ.csv -sod 3 -sot LocalTimestamp -soa AX AY AZ -stf .\cyton5.txt -std 3 -stt " Timestamp" -sta " Accel Channel 0" " Accel Channel 1" " Accel Channel 2" -n1 EmotiBit -n2 Cyton -o taps5 -w2 13 -h2 0.08```
```

Comment:
I wonder if it helps to split this command and explain piecewise "what each argument is doing" and "we we need it" for our sample example.
something like:
-sod 3 is used because ` - - - `

My only reason against it would be that it might be duplicating the information you can get from `-h` CLI option. What do you think about that?

Reply:
It sort of duplicates it, but I think it would be a good idea to include it so that it can be seen within the context of arguments that are provided. I'll add it, but if you think it is overkill after seeing what that looks like it, I can always take it back out.